common/src/main/java/io/netty/util/collection/PrimitiveCollections.java

```diff
@@ -50,6 +52,23 @@ private PrimitiveCollections() {
      * An empty map. All operations that attempt to modify the map are unsupported.
      */
     private static final class EmptyIntObjectMap implements IntObjectMap<Object> {
+        // TODO: replace with Collections.emptyIterator when we drop support for Java < 1.7.
+        private static final Iterator<Entry<Integer, Object>> iterator = new Iterator<Entry<Integer, Object>>() {
```

Comment:
`iterator` -> `EMPTY_ITERATOR`?

Also is this needed anymore?  If so should we expose it via a `public static` method of some kind?